app/code/Magento/Cms/view/adminhtml/ui_component/cms_block_listing.xml

```diff
@@ -146,7 +146,6 @@
                 <editor>
                     <validation>
                         <rule name="required-entry" xsi:type="boolean">true</rule>
-                        <rule name="validate-xml-identifier" xsi:type="boolean">true</rule>
```

Comment:
What server-side validation is being used? Just required-entry? Just making sure frontend is consistent with serverside validation

Reply:
Hello @erfanimani ,

As Per Block add/edit in details page there is same validation.
That's Why i am removing another validation and add only require validation.

Please check attached screen shot
https://nimb.ws/VNP5Rz
https://nimb.ws/Rkzf2A

There is no validation for special character space and any other. there is only required validation is available.

Reply:
Hello @erfanimani ,

As Per Block add/edit in details page there is same validation.
That's Why i am removing another validation and add only require validation.

Please check attached screen shot
https://nimb.ws/VNP5Rz
https://nimb.ws/Rkzf2A

There is no validation for special character space and any other. there is only required validation is available.

Reply:
Perfect — I was just checking :)

Thank you